jumpscale/clients/taiga/taiga.py

```diff
@@ -217,3 +217,190 @@ def _get_project_status(project_statuses, status):
 
         self.api.user_stories.delete(story_id)
         return migrate_story.id
+
+    def list_all_issues(self, user_id=""):
+        """
+        List all issues for specific user if you didn't pass user_id will list all the issues
+        
+        Args:
+            user_id (int): id of the user.
+
+        Returns:
+            List: List of issues.
+        """
+        return self.api.issues.list(assigned_to=user_id)
+
+    def list_all_projects(self):
+        """
+        List all projects
+        
+        Returns:
+            List: List of projects.
+        """
+        return self.api.projects.list()
+
+    def list_all_milestones(self):
+        """
+        List all milestones
+        
+        Returns:
+            List: List of milestones.
+        """
+        return self.api.milestone.list()
+
+    def list_all_user_stories(self, user_id=""):
+        """
+        List all user stories for specific user if you didn't pass user_id will list all the available user stories
+        
+        Args:
+            user_id (int): id of the user.
+            
+        Returns:
+            List: List of user stories.
```

Comment:
the same as https://github.com/threefoldtech/js-sdk/pull/1355/files#r499798935 too.